### PR TITLE
Added test for CallMeBackController

### PR DESCRIPTION
**Pull request by kebrahim**
- [x] Tests pass
- [x] Appropriate changes to README are included in PR
- [X] Feature has been fleshed out in design document

### CODE REVIEW

File: callmeback/src/test/java/org/google/callmeback/CallMeBackControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+@SpringBootTest
```
Comment (egracer) [inline callmeback/src/test/java/org/google/callmeback/CallMeBackControllerTest.java:13]: If all that you want to do is test that these HTTP methods exist, better not to use this annotation. See the @WebMvcTest and @MockBean annotations as described here: https://spring.io/guides/gs/testing-web/

Comment (kebrahim) [reply callmeback/src/test/java/org/google/callmeback/CallMeBackControllerTest.java]: done

File: callmeback/src/test/java/org/google/callmeback/CallMeBackControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+@WebMvcTest
+public class CallMeBackControllerTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @MockBean
+    private MappingMongoConverter converter;
+    @MockBean
+    private IsNewAwareAuditingHandler mongoAuditingHandler;
```
Comment (egracer) [inline callmeback/src/test/java/org/google/callmeback/CallMeBackControllerTest.java:23]: Curious how you discovered that these were the beans that you needed to Mock. Did you just run the test with the MockMvc and see what errors were thrown?

Comment (kebrahim) [reply callmeback/src/test/java/org/google/callmeback/CallMeBackControllerTest.java:23]: yes, that's exactly what i did :)

**Pull request merged**